src/tests/ftest/dfuse/root_container.py

```diff
@@ -142,21 +143,28 @@ def verify_create_delete_containers(self, pool, device, cont_count, hosts, mount
         pool_space_after = pool.get_pool_free_space(device)
         self.log.info("Pool space <= Expected")
         self.log.info("%s <= %s", pool_space_after, expected)
-        self.assertTrue(pool_space_after <= expected)
+        self.assertTrue(
+            pool_space_after <= expected,
+            f"Failed pool free space check after creating {cont_count} containers")
 
         self.log_step(f"Destroy half of the {cont_count} new sub containers ({cont_count // 2})")
         for _ in range(cont_count // 2):
             containers[-1].destroy(1)
             containers.pop()
 
+        self.log.info("Wait 60 seconds before checking for free space gain")
+        time.sleep(60)
```

Comment:
I know this was a suggestion from triage but FYI this is just arbitrary and could take longer.